src/components/Navbar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+function Navbar() {
+  return (
+    <div className="bg-[#280cb3] text-white border-b border-gray-950">
+      <div className="flex justify-between items-center px-4 py-3">
+        <div className="text-lg font-bold">My App</div>
+        
+        <ul className="hidden sm:flex space-x-6 text-base font-medium">
+          <li>
+            <a href="/" className="hover:text-amber-400">Home</a>
```

Comment:
Name routes in all small. If there are more than one words, we do something like to-do or about-us.